src/Storage.Interface/Models/Application.cs

```diff
@@ -110,6 +110,13 @@ public class Application : ChangableElement
 
         public CopyInstanceSettings CopyInstanceSettings { get; set; }
 
+        /// <summary>
+        /// Gets or sets an optional alternate number to append to the storage container name
+        /// </summary>
+        [JsonProperty(PropertyName = "alternateContainerNumber")]
+
+        public int? AlternateContainerNumber { get; set; }
```

Comment:
Maybe call it StorageContainerNumber? More informative for someone who is just looking at the application metadata.